app/models/user.rb

```diff
@@ -1,7 +1,10 @@
 class User < ApplicationRecord
-  has_many :likes, foreign_key: 'author_id'
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   
```

Comment:
- [ ] It would be nice if you make unit tests for these validations to make your project follow the requirements 👍